SetDemo/src/SetDemo.java

```diff
@@ -0,0 +1,24 @@
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.Set;
+
+public class SetDemo {
+    public static void main(String[] args) {
+
+        LinkedList<String> list=new LinkedList<>();
+        list.add("Petr");
+        list.add("Anna");
+        list.add("Maria");
+        list.add("Ivan");
+        list.add("Anna");
+        list.add("Petr");
+        list.add("Filip");
+        list.add("Anna");
+        list.add("George");
+
+        //System.out.println(list);
+        Set<String> set= new HashSet<>();//inner class?
+
+        set.addAll(list);// Set allows no duplicates, consequently they will be eliminated by adding them to a set
+        System.out.println(set);
```

Comment:
Definition of done: the task is solved as a method, which accepts some parameters and returns a result. There is a test, which proves that the method is correct for some basic use cases. 

Your solution is just about showing that you know how to write some code using Set API. It is not enough.